src/main/java/net/sf/mzmine/modules/peaklistmethods/io/gnpsexport/GNPSExportParameters.java

```diff
@@ -36,4 +41,15 @@
     public GNPSExportParameters() {
     	super(new Parameter[] {PEAK_LISTS, FILENAME, MASS_LIST});
     }
-}
+    
+    public ExitCode showSetupDialog(Window parent, boolean valueCheckRequired) { 
+    	String message = "<html>GNPS Module Disclaimer:" + 
+        		"<br>    - If you use the GNPS export module for GNPS web-platform, cite MZmine2 paper and the following article:"+
+        		"<br>     Wang et al., Nature Biotechnology 34.8 (2016): 828-837." +
+        		"<br>    - See the documentation about MZmine2 data pre-processing for GNPS (<a href=\"\">http://gnps.ucsd.edu/</a>) molecular " +
+        		"<br>     networking and MS/MS spectral library search. <a href=\"\">https://bix-lab.ucsd.edu/display/Public/GNPS+data+analysis+workflow+2.0</a></html>";
```

Comment:
The reason is because the html-tag provides the visual of a link, but regardless of whether or not href is populated, the label does not implement a listener to redirect to a website.